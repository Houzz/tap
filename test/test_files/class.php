<?php

class ClassA {
  /**
   * @var float
   */
  public $p1, $p2;

  /**
   * @param int $a
   * @param string $b
   * @return bool
   */  
  public function methodA($a, $b) {
      $c = $a + 1;
      return $c;    // wrong return type
  }
}

