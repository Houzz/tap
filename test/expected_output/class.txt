16 E_RETURN_TYPE
